plaidml/bridge/tensorflow/tests/BUILD

```diff
@@ -143,7 +146,10 @@ tf_cc_test(
     data = [":i3d_test_genrule"],
     extra_copts = ["-fexceptions"],
     linkstatic = 1,
-    tags = ["skip_windows"],
+    tags = [
+        "manual",
```

Comment:
@dgkutnic I made this test `manual` for now until we can either:
1. Get the model frozen before we convert it to HLO, so that all the existing variables come in as `Constant()` rather than `Placeholder()`.
2. Update the arguments being passed to the Executable so that the weights are in the correct order. This seems like lots of tedious work, and it's actually pretty brittle; we shouldn't require users to have to know the correct order.

I've made some progress on figuring out how to freeze the graph, we can chat later about it.